Sources/SwiftDocC/Model/Rendering/RenderNode/RenderMetadata.swift

```diff
@@ -144,6 +144,15 @@ public struct RenderMetadata: VariantContainer {
     /// The variants for the source file URI of a page.
     public var sourceFileURIVariants: VariantCollection<String?> = .init(defaultValue: nil)
     
+    /// The source location of the topic's source code.
```

Comment:
```suggestion
    /// The source location of the topic's declaration.
```

Reply:
I renamed this property and updated the documentation comment:

```swift
/// The remote location where the source declaration of the topic can be viewed.
public var remoteSource: RemoteSource? {
```